### DISCUSSION TITLE
Is it possible to exclude a group name from the GWSGroupsFilter?

### DISCUSSION THREAD

**Discussion by tmylonidis** [slashdevops/idp-scim-sync#72] [Q&A] [answered]
Hi! Do you know if it is possible to exclude a group name from the GWSGroupsFilter when deploying the application? Assuiming that our current GWSGroupsFilter is 'name:AWS*' is there any way that we can have an exception?

Also, can the GWSGroupsFilter be altered after the solution is deployed using the Serverless Application Repository? Or is it necessary to delete the stack and redeploy again?

Thanks!

**Comment by christiangda** [comment, marked as answer]
Hi @tmylonidis,
The filter is applied in the source of truth (I mean google workspace) not in the runtime.  I recommended you to read this [Using SSO
](https://github.com/slashdevops/idp-scim-sync/blob/main/docs/Using-SSO.md).

But in any case, you can use any attribute in the group to filter these, not only the `'name:AWS*'` if you read the documentation related maybe you can find a way to filter better your groups.

read the documentation here [Search for groups](https://developers.google.com/admin-sdk/directory/v1/guides/search-groups)

Also, remember that you have available the program `idpscimcli` and their commands `idpscimcli gws groups --help` to test your filters.

To use this, just `download the binary from the release` or `clone the repo` and execute `make` then the program will be inside `build/` folder.
